subgraphs/_reference_/src/sdk/protocols/lending/manager.ts

```diff
@@ -227,6 +230,10 @@ export class DataManager {
     return this.market;
   }
 
+  isNewMarket(): boolean {
```

Comment:
will this not always return true? `newMarket` is set to true on creation of a new market, then never updated. :thinking: 

should it be,
```
constructor(...) {
  let _market = Market.load(marketID);
  this.newMarket = false;
  if (!_market) {
    ...
    this.newMarket = true;
  }
}
```

Reply:
That is what is happening. Defaults to false, but if a new market is made it is set to true